pyproject.toml

```diff
@@ -48,8 +48,8 @@ test = [
 
 docs = [
     "volatility3[dev]",
-    "sphinx>=8.0.0,<7",
-    "sphinx-autodoc-typehints>=2.5.0,<3",
```

Comment:
I assume this change is to work with the version of sphinx available?  Again, I believe the lower bound was pulled up artificially when the `pyproject.toml` inhereited the values from the `requirements.txt`.